packages/expo-device/package.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "expo-device",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.1",
```

Start with a version number that communicates this is a preview until it's ready for 1.0.